app/templates/components/forms/orders/attendee-list.hbs

```diff
@@ -14,6 +14,11 @@
             <i class="user icon"></i>
             <label>{{t 'Ticket Holder '}}{{t ' -for- '}}{{holder.ticket.name}}</label>
           </div>
+          <button class="ui labeled icon {{if this.device.isMobile 'fluid'}} button"
+          {{action this.downloadTicketForAttendee holder.event.name holder.order.id holder.id }}>
```

Comment:
@maze-runnar  - here the holder.event.name when downloading ticket is undefined, but when i print in console, it gives me the value of the event. Please help

Reply:
Also how do I align to end this button?
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/20026440/99994812-878c2e00-2ddf-11eb-8a5c-18e4c96b30ba.png">

Reply:
> Also how do I align to end this button?
> <img alt="image" width="1437" src="https://user-images.githubusercontent.com/20026440/99994812-878c2e00-2ddf-11eb-8a5c-18e4c96b30ba.png">

`<button class="ui right floated button">Right Floated</button>`

Reply:
> @maze-runnar - here the holder.event.name when downloading ticket is undefined, but when i print in console, it gives me the value of the event. Please help

`@event.name` for event name.

Reply:
Can you also xplain @maze-runnar plz

Reply:
in order/view.hbs

```
<Forms::Orders::AttendeeList
        @save="save"
        @data={{this.model.order}}
        @fields={{this.model.form}}
        @event={{this.model.event}} />
```